File: sbt-test/android-sdk-plugin/hello-multidex/src/main/scala/com/example/app/MainActivity.scala
package com.example.app

import android.app.Activity
import android.os.Bundle
import android.util.Log

class MainActivity extends Activity {
  override def onCreate(bundle: Bundle):Unit = {
    super.onCreate(bundle)
    setContentView(R.layout.hello)
    val foo = "foo"
    Log.v("MainActivity", s"interpolationTest=$foo")
  }
}

File: sbt-test/android-sdk-plugin/simple-retrolambda/src/main/java/com/example/retrolambda/MainActivity.java
package com.example.retrolambda;

import android.app.Activity;
import android.os.Bundle;

import com.google.common.collect.*;
import com.google.common.base.*;
import java.util.*;

public class MainActivity extends Activity
{
    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
    }

    public static List<Integer> toIntList(List<Character> characters) {
        return Lists.transform(characters,  c -> (int) c.charValue());
    }
}
